resources/data/contributors.json

```
@@ -992,6 +992,23 @@
           38.00278
         ]
       }
+    },
+    {
+      "type": "Feature",
+      "properties": {
+        "Name": "Vincent Cloarec",
+        "Committer": "No",
```

```suggestion
        "Committer": "Yes",
```
Was this not accepted?

---

best to ask @mbernasocchi  , AFAIK it was postponed until discussed on PSC, see https://lists.osgeo.org/pipermail/qgis-psc/2020-May/008768.html